src/includes/sentry-cli-sourcemaps.mdx

```diff
@@ -18,28 +16,10 @@ Visit the [auth token user settings page](https://sentry.io/settings/account/api
 
 </Note>
 
-```bash
-export SENTRY_AUTH_TOKEN=your-auth-token
-export SENTRY_ORG=___ORG_SLUG___
-export SENTRY_PROJECT=___PROJECT_SLUG___
-```
-
-## 2. Generate Source Maps
-
-Configure the bundler to output source maps. Here's an example of what this looks like for Webpack:
-
-```javascript {filename:webpack.config.js}
-module.exports = {
-  ...
-  entry: "./src/index.js",
-  output: {
-    filename: "main.js",
-    path: path.resolve(__dirname, "build"),
-  },
-  // Enable "sourcesContent".
-  devtool: "source-map",
-  ...
-};
+```{filename:.env.local}
```

Comment:
Let's re-add bash so that formatting and highlighting is fixed
```suggestion
```bash {filename:.env.local}
```